_automating-configurations/api/get-workflow-steps.md

```diff
@@ -28,36 +25,52 @@ OpenSearch validates workflows by using the validation template that lists the r
     ]
   }
 }
-```
-
-The Get Workflow Steps API retrieves this file.   
+``` 
 
 ## Path and HTTP methods
 
 ```json
 GET /_plugins/_flow_framework/workflow/_steps
+GET /_plugins/_flow_framework/workflow/_step?workflow_step=<step_name>
 ``` 
 
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| `workflow_step` | String | The step name of the step to retrieve. Specify multiple step names as a comma-separated list. For example, `create_connector,delete_model,deploy_model`. |
+
 #### Example request
 
+To fetch all workflow steps, use the following request:
+
 ```json
 GET /_plugins/_flow_framework/workflow/_steps
+``` 
+{% include copy-curl.html %}
+
+To fetch specific workflow steps, pass the step names to the request as a query parameter:
+
+```json
+GET /_plugins/_flow_framework/workflow/_step?workflow_step=create_connector,delete_model,deploy_model
 ```
 {% include copy-curl.html %}
 
 
 #### Example response
 
-OpenSearch responds with the validation template containing the steps. The order of fields in the returned steps may not exactly match the original JSON but will function identically.
+OpenSearch responds with the workflow steps. The order of fields in the returned steps may not exactly match the original JSON but will function identically.
```

Comment:
Should a noun follow JSON?

Reply:
I don't think so, I think it's understood that JSON means the "document in JSON format".